doctr/models/detection/fast/tensorflow.py

```diff
@@ -0,0 +1,372 @@
+# Copyright (C) 2021-2024, Mindee.
+
+# This program is licensed under the Apache License 2.0.
+# See LICENSE or go to <https://opensource.org/licenses/Apache-2.0> for full license details.
+
+# Credits: post-processing adapted from https://github.com/xuannianz/DifferentiableBinarization
```

Comment:
Mh the post processing from FAST is not useful in our case (because the model was made for Scene text detection in the wild and it simply extracts the contours from the prob_map this will not work for images with lots of close text) So this should be fine it's the post proc from DB (a bit modified)

Reply:
OK !